docs/en/user_guides/classification.md

```diff
@@ -46,14 +45,23 @@ Remarks:
 
 ## Linear Evaluation
 
-The linear evaluation is one of the most general benchmarks, we integrate several papers' config settings, also including multi-head linear evaluation. We write classification model in our own codebase for the multi-head function, thus, to run linear evaluation, we still use `.sh` script to launch training. The supported datasets are **ImageNet**, **Places205** and **iNaturalist18**.
+The linear evaluation is one of the most general benchmarks, we integrate several papers' config settings, also including multi-head linear evaluation. We write a classification model in our own codebase for the multi-head function, thus, to run linear evaluation, we still use `.sh` script to launch training. The supported datasets are **ImageNet**, **Places205** and **iNaturalist18**.
+
+First, make sure you have installed [MIM](https://github.com/open-mmlab/mim), which is also a project of OpenMMLab.
+
+```shell
+pip install openmim
+```
+Besides, please refer to MMClassification for [installation](https://github.com/open-mmlab/mmclassification/blob/dev-1.x/docs/en/install.md) and [data preparation](https://github.com/open-mmlab/mmclassification/blob/dev-1.x/docs/en/getting_started.md).
+
+Then, run the command below.
 
 ```shell
 # distributed version
-bash tools/benchmarks/classification/dist_train_linear.sh ${CONFIG} ${PRETRAIN}
+bash tools/benchmarks/classification/mim_dist_train.sh ${CONFIG} ${PRETRAIN}
 
 # slurm version
-bash tools/benchmarks/classification/slurm_train_linear.sh ${PARTITION} ${JOB_NAME} ${CONFIG} ${PRETRAIN}
+bash tools/benchmarks/classification/mim_slurm_train.sh ${PARTITION} ${JOB_NAME} ${CONFIG} ${PRETRAIN}
 ```
```

Comment:
Better to give specific examples of the command

Reply:
give examples of training and tests for cls, det, and seg